R/utilities.R

```diff
@@ -805,181 +805,414 @@ trim_scale_expression = function(x){
   return(x)
 }
 
+#' @title Process Regions objects.
+#'
+#' @description INTERNAL FUNCTION to harmonize genomic regions specified as character vectors or data frames.
+#'
+#' @details INTERNAL FUNCTION to harmonize genomic regions specified as character vectors or data frames.
+#'
+#' @param regions Character vector of genomic regions. If neither regions nor regions_df is specified, will use GAMBLR aSHM regions
+#' @param regions_df Data frame of genomic regions with column names "chrom", "start", "end", "name"
+#' @param region_padding Amount to pad the start and end coordinates by. [0]
+#' @param skip_regions Character vector of genes to drop from GAMBLR aSHM regions.
```

Comment:
the skip and keep parameters seem specific to the aSHM region list. I think that's fine but we should probably be explicit in documenting that for the user. regions_df should also be documented as optional and we should specify that the default is ashm_regions for the specified projection.

Reply:
```If neither regions nor regions_df is specified, will use GAMBLR aSHM regions```
Is that clear that these are optional and what the default is? 
```Character vector of genes to drop from GAMBLR aSHM regions.```
How would you change this to make it clearer that this applies to the default?

Reply:
Oh I didn't see that, probably because I was focusing on regions_df. Maybe just duplicate that comment on the regions_df line?